pkg/syncer/metadatasyncer.go

```diff
@@ -390,27 +397,26 @@ func ReloadConfiguration(ctx context.Context, metadataSyncer *metadataSyncInform
 				log.Debugf("Unregistering virtual center: %q from virtualCenterManager", metadataSyncer.configInfo.Cfg.Global.VCenterIP)
 				err = vcManager.UnregisterAllVirtualCenters(ctx)
```

Comment:
Why dont you add `UnregisterAllVirtualCenters` to `GetVirtualCenterInstance` so that the caller(s) don't have to deal with it?

Reply:
I have moved the function to `GetVirtualCenterInstance`. One question here - is it possible to have multiple virtual centers registered with `vcManager`. If yes, why are we unregistering all the VCs? @divyenpatel

Reply:
We dont support clusters across VCs as of today, so i think its fine